targeteditor/src/main/scala/explore/targeteditor/CoordinatesForm.scala

```diff
@@ -59,7 +59,7 @@ object CoordinatesForm {
 
   @Lenses
   final case class State(
-    searchTerm:  String,
+    searchTerm:  NonEmptyString,
```

Comment:
I didn't want to use eariler `NonEmptyString` as an empty input is valid though it is useless to do a search. I suppose with this change (validating in `FormInputEV`) we still let an empty string but we can't do a search?

Reply:
Exactly, we use the new `FormInputEV` validation capabilities to trigger the error message. I will add another commit to  disable the search button when input is invalid (string empty) with this new logic.